Adding benchmarks (#29)
Optable/match: README.md

A complete test suite for all PSIs is present [here](test/psi). Don't hesitate to take a look and help us improve the quality of the testing by reporting problems and observations!

# benchmarks

> A benchmark on the running times of different PSI protocols of this library is provided [here](benchmark/README.md).

Reviewer: See runtime benchmarks of the different PSI protocols [here](benchmark/README.md).

> A comparison between running times of different PSI protocols of this library is provided [here](benchmark/README.md).

Reviewer 2: ```suggestion
A benchmark on the running times of different PSI protocols of this library is provided [here](benchmark/README.md).
```
